libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -124,11 +124,38 @@ AP_Vehicle& vehicle = *AP_Vehicle::get_singleton();
 extern AP_Vehicle& vehicle;
 #endif
 
+#if defined(HAL_GPIO_PIN_WDO)
+static int cnt = 0;
+static bool _sw = false;
+void AP_Vehicle::timer_update(void)
+{
+    if (cnt == 50)
+    {
+        cnt = 0;
+    	if (_sw == true)
+	    {
```

Comment:
ardupilot has a minimal coding style.

https://ardupilot.org/dev/docs/style-guide.html

Reply:
I fixed it by running this program.